conf/messages

```diff
@@ -1057,6 +1057,12 @@ multipleDisposalsWereAllPropertiesResidential.details.li3=land that’s part of
 
 multipleDisposalsWereAllPropertiesResidential.error.required=Select yes if all the properties were residential properties
 
+countryCode.title=What was your country of residence on the day you ‘exchanged’ contracts?
```

Comment:
perhaps something like `multipleDisposalsCountryCode` would be a better scope here